pkg/formula/runner/inputs.go

```
@@ -246,3 +250,43 @@ func (in InputManager) resolveIfReserved(input formula.Input) (string, error) {
 	}
 	return "", nil
 }
+
+func (in InputManager) verifyConditional(cmd *exec.Cmd, input formula.Input) bool {
+	if input.Condition.Variable == "" {
+		return true
+	}
+
+	var value string
+	variable := input.Condition.Variable
+	for _, envVal := range cmd.Env {
+		components := strings.Split(envVal, "=")
+		if strings.ToLower(components[0]) == variable {
+			value = strings.ToLower(components[1])
+			break
+		}
+	}
+	if value == "" {
+		message := fmt.Sprintf("config.json: Conditional variable %s not found. Showing prompt item anyway.", variable)
+		prompt.Warning(message)
+		return true
+	}
+
+	// Currently using case implementation to avoid adding a dependency module or exposing
+	// the code to the risks of running an eval function on a user-defined variable
+	// optimizations are welcome, being mindful of the points above
+	switch input.Condition.Operator {
+	case "==": return value == input.Condition.Value
+	case "!=": return value != input.Condition.Value
+	case ">":  return value > input.Condition.Value
+	case ">=": return value >= input.Condition.Value
+	case "<":  return value < input.Condition.Value
+	case "<=": return value <= input.Condition.Value
+	default:
+		message := fmt.Sprintf(
```

you should return an error when the operator is invalid, because the fallback to equality may not be clear to the user.
Taking into account programming languages, if you pass an invalid operator your code will not compile.